framework/src/main/java/org/apache/drill/test/framework/Utils.java

```diff
@@ -169,7 +170,21 @@ public static ConnectionPool getConnectionPool() {
         boolean foundTests = false;
         for (String testGroup : testGroups) {
           if (categories != null && !categories.contains(testGroup)) {
-            continue;
+            if (categories.contains("smoke") &&
```

Comment:
The main concern I have is, how do we know if we missed simba_jdbc somewhere.  I think it would be better to run with it "everywhere" and disable it when necessary.  Rather than add it when we remember to add it, which means we may miss it in some places.